docs_user/modules/proc_adopting-networker-services-to-the-data-plane.adoc

```diff
@@ -305,39 +287,35 @@ spec:
   - openstack-networker
 EOF
 ----
-
-Note: alternatively, you could include the `Networker` specific node set in the
-`nodeSets` list when creating the main, `openstack`,
-`OpenStackDataPlaneDeployment` CR.
-
-Note that once a `OpenStackDataPlaneDeployment` CR is deployed, it's impossible
-to add new node sets to it later.
++
+[NOTE]
+Alternatively, you can include the `Networker` specific node set in the
+`nodeSets` list when you create the main `openstack` `OpenStackDataPlaneDeployment` CR, before you deploy it. Note that once an `OpenStackDataPlaneDeployment` CR is deployed, you cannot add new node sets to it later.
 
 .Verification
 
 . Confirm that all the Ansible EE pods reach a `Completed` status:
 +
 ----
 # watching the pods
-watch oc get pod -l app=openstackansibleee
+$ watch oc get pod -l app=openstackansibleee
 ----
 +
 ----
 # following the ansible logs with:
-oc logs -l app=openstackansibleee -f --max-log-requests 20
+$ oc logs -l app=openstackansibleee -f --max-log-requests 20
 ----
 
 . Wait for the data plane node set to reach the `Ready` status:
 +
 ----
-oc wait --for condition=Ready osdpns/openstack-networker --timeout=30m
+$ oc wait --for condition=Ready osdpns/openstack-networker --timeout=30m
```

Comment:
Is `osdpns/openstack-networker` the name of a node set? Should users replace this value with their own?

Reply:
the name is `openstack-networker` user should replace only this part

Reply:
I suggest changing `openstack-networker` to `<networker_CR_name` (if accurate).

"* Replace `<networker_CR_name>` with the name of the CR that you deployed for your Networker nodes, for example, `openstack-networker`."

Reply:
@fao89 Do you have any objections to my proposed rewrite, above?